dumbo/src/tcp/connection.rs

```diff
@@ -1651,4 +1656,11 @@ pub(crate) mod tests {
         // and we don't wait for our FIN to be ACKed.
         assert!(c.is_done());
     }
+
+    #[test]
+    fn test_xor_rng_u32() {
```

Comment:
This test is not really useful, because 1000 such comparisons don't necessarily have be `true`. I think we should drop this altogether, unless we're testing some meaningful statistical property of `xor_rng_u32`.